enums.rb

```diff
@@ -0,0 +1,221 @@
+# rubocop:disable Style/For
+# rubocop:disable Metrics/ModuleLength
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength
+module Enumerable
+  def my_each
+    if block_given?
+      for element in self
+        yield(element)
+      end
+    else
+      to_enum
+    end
+  end
```

Comment:
* [ ] should have working `my_each` identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
```ruby
  my_each_output = ''
  each_output = ''

  [1,2,3,4,5,6].my_each { |num| my_each_output += num.to_s }
  [1,2,3,4,5,6].each { |num| each_output += num.to_s }

  my_each_output != each_output
```

Reply:
Hello! I have verified this output and this one is the same :) 
![image](https://user-images.githubusercontent.com/53732843/81595227-5c7c4880-937f-11ea-87ba-2f834156654c.png)

Reply:
You should verify by comparing `my_each_output` and `each_output`.

Reply:
Outputs are the same please Gladly check 👍 
![image](https://user-images.githubusercontent.com/53732843/81597630-42dd0000-9383-11ea-92a8-050b5d02b54b.png)